### PR TITLE
Add blue-green deploys.

### PR DESCRIPTION
**Pull request by jmcarp**
Code borrowed from API.

### CODE REVIEW

**Comment by jmcarp** [comment]
Important: this deploy uses two applications, "web-a" and "web-b". Environment variables must be set for these new apps in order for them to work--otherwise, we'll get errors. @arowla let's talk about how to proceed before deploying.

**Comment by arowla** [comment]
Let's set up the user-provided service. I've seen enough of hand-set ENV vars disappearing already, even without this change.

**Comment by arowla** [comment]
Issue 18F/openFEC#800

**Comment by arowla** [comment]
See 18F/openFEC#796

File: manifest_prod.yml [hunk 1]
```diff
@@ -6,10 +6,25 @@ env:
...
   path: .
   buildpack: "https://github.com/ddollar/heroku-buildpack-multi.git"
   host: fec-prod-web
+  services:
+    - fec-creds-prod
+  env:
+    NEW_RELIC_APP_NAME: OpenFEC Web (production)
+    FEC_WEB_API_URL: https://api.open.fec.gov/
+    FEC_FORCE_HTTPS: true
+    # These must be set manually via `cf set-env`
```
Comment (arowla) [inline manifest_prod.yml]: Would it make sense to note somewhere (or just right here) all the additional varnames we now need to make sure are manually set on the creds service?

```
# FEC_WEB_API_KEY
# FEC_WEB_API_KEY_PUBLIC
# NEW_RELIC_LICENSE_KEY
```

... And any others I might have missed.

Comment (jmcarp) [inline manifest_prod.yml]: :+1:

**Pull request merged**